.github/ISSUE_TEMPLATE/ad-hoc-meeting.yaml

```diff
@@ -0,0 +1,51 @@
+name: 'Create an ad-hoc meeting'
+description: 'This issue template is used for creating a new ad-hoc meeting for the Cross Project Council'
+title: "[Ad hoc meeting] "
+assignees:
+  - bensternthal
+labels:
+  - cross-project-council-agenda
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Agenda for the Meeting
+  - type: input
+    id: facilitator
+    attributes:
+      label: Facilitator
+      description: Who is the facilitator of this meeting?
+    validations:
+      required: true
+  - type: input
+    id: related-issues
+    attributes:
+      label: Related issue(s)
+      description: What are the related issues to this meeting?
+    validations:
+      required: false
+  - type: input
+    id: meeting-date
+    attributes:
+      label: Date and time
+      description: 'When should the meeting happen? Use a "YYYY-MM-DD HH:MM UTC" format (e.g.: 2009-05-27 13:00 UTC)'
```

Comment:
Nit, my preference is that this is NOT UTC but whatever timezone the requester is in. IMHO this is easier for the requestor and whoever is fulfilling this request.